docs/source/VUDRM-token.md

```diff
@@ -39,13 +39,18 @@ The DRM policy is included in the VUDRM token as the encrypted third component.
 | `duration_rental`  | string   | Seconds                   | Fairplay       | Overrides polend for Fairplay rental licenses.                                                               |
 | `duration_lease`   | string   | Seconds                   | Fairplay       | Overrides polend for Fairplay lease licenses.                                                              |
 | `duration_persist` | string   | Seconds                   | Fairplay       | Overrides polend for Fairplay persist licenses.                                                           |
+| `match_content_id` | bool     | `true` or `false`         | Fairplay, Widevine       | Boolean for if the content id in the token should be compared with the content id used to encrypt a piece of content.                                                     
```

Comment:
Not sure on the wording of this... 
The content id will be matched to the content id in the license request, which ultimately is the content id used when generating the encryption keys.

I think it needs to be more explicit?